### PR TITLE
Fix #346

### LINKED ISSUE #346: Search field for contributor selection
The current contributor selection is a simple dropdown list with all users. Therefore you can only search for people when starting to type their first name.
This should be changed so that the dropdown is replaced by an edit field that has a dropdown option which can be manually triggered and that opens automatically when starting to type and which then only holds names that contain the typed substring (like the multiselect input alread does).

### PR DESCRIPTION
**Pull request by lawrencebenson**
changed contributor selection from static drop down select to input
based drop down

### CODE REVIEW

File: evap/contributor/templates/contributor_course_form.html [hunk 1]
```diff
@@ -34,6 +34,7 @@
                 </thead>
                 <tbody>
                     {% for form_element in formset %}
+
```
Comment (Bartzi) [inline evap/contributor/templates/contributor_course_form.html]: could you remove this line?

File: evap/contributor/templates/contributor_course_form.html [hunk 2]
```diff
@@ -46,11 +46,17 @@
...
                             <td>
                                 {{form_element.contributor.errors}}
+                                
                                 {% if form_element.responsible.value == True %}
-                                    <select class="form-control" disabled="True"><option selected="selected">{{ responsible }} ({% trans "responsible" %})</option></select>
+                                    <input value='{{ responsible }} ({% trans "responsible" %})' disabled='True'/>
                                     {{ form_element.contributor.as_hidden }}
                                 {% else %}
-                                    {{ form_element.contributor }}
+                                    <input list="contributor_list" style="width:80%"/>
```
Comment (Bartzi) [inline evap/contributor/templates/contributor_course_form.html]: could you please add the css class form-control to each input?

**Comment by Bartzi** [comment]
* could you have a look at the tests and why they fail?
* could you also add the functionality you implemented to the "course_edit" view in "/staff/views.py"?
you can find it here: 
https://github.com/fsr-itse/EvaP/blob/master/evap/staff/views.py#L307

**Comment by karyon** [comment]
will be superseded by select2/chosen in #529.

**Pull request closed without merging**